bodhi/server/models.py

```diff
@@ -1339,6 +1454,36 @@ class ModuleBuild(Build):
         'polymorphic_identity': ContentType.module,
     }
 
+    @property
+    def nvr_name(self):
+        """
+        Return the ModuleBuild's name.
+
+        Returns:
+            str: The name of the module.
+        """
+        return self._get_kojiinfo()['name']
+
+    @property
+    def nvr_version(self):
+        """
+        Return the the ModuleBuild's stream.
+
+        Returns:
+            str: The stream of the ModuleBuild.
+        """
+        return self._get_kojiinfo()['version']
+
+    @property
+    def nvr_release(self):
+        """
+        Return the ModuleBuild's version.
```

Comment:
Would the version also include the context? If so, might be good to mention here too.